### DISCUSSION TITLE
Line and Paragraph Spacing for DocX

### DISCUSSION THREAD

**Discussion by ReReNo1** [eclipse-birt/birt#1435] [Q&A]
Hi there

I am creating a BIRT report to work with SuccessFactors and the out is in Word (Docx), the issue is I need to add line and paragraph spacing to some of the text boxes within the boxes of the table, but can not see how I can get this to show as required.  Would appreciate if someone could advise if this is possible please?
 
![Spacing Requirements](https://github.com/eclipse-birt/birt/assets/146188922/945b87db-c2d0-4ff2-b5aa-146ccd3d8b59)

TIA
Regards
Rehanna

**Comment by hvbtup** [comment]
You can set the line spacing in the advanced properties.

You cannot set paragraph spacing.
But if each paragraph is a an individual layout item, you can use the box properties (margin-top, padding-top, margin-bottom, padding-bottom to achieve a similar effect.

**Reply by ReReNo1** [reply]
Thanks @hvbtup 

Can you please advise which attributes in the advanced settings would allow for me to set the line spacing Multiple at 1.1, I have tried a number of settings, within advanced settings, but I can't get the Multiple line spacing to work

![image](https://github.com/eclipse-birt/birt/assets/146188922/d658be60-721f-4c4e-8650-f0bb29a86c5c)

Thanks

**Reply by hvbtup** [reply]
Line Height
